src/lib/VncScreen.tsx

```diff
@@ -113,6 +113,16 @@ const VncScreen: React.ForwardRefRenderFunction<VncScreenHandle, Props> = (props
         connected.current = state;
     };
 
+    const refocusCanvasOnCaptureElem = () => {
+        const canvas = document.querySelector('canvas');
```

Comment:
If there's another `canvas` on the page before the `noVNC` canvas, this line will select that won't it?

Maybe we could do something like
```ts
const canvases = Array.from(document.querySelectorAll('canvas'));
const canvas = canvases.filter(({ id }) => id === 'noVNC_mouse_capture_elem');
```

By the way, have you tested this approach? It does feel a little hacky, but since a lot of people have been facing it I'm happy to merge it (provided it doesn't break anything of course 😅)